Assets/Thirdweb/Core/Scripts/Transaction.cs

```diff
@@ -0,0 +1,254 @@
+                throw new UnityException("This functionality is not yet available on your current platform.");
+                // string route = contract.abi != null ? $"{contract.address}{Routable.subSeparator}{contract.abi}" : contract.address;
+                // string sendRoute = $"{route}{Routable.separator}send";
+                // return await Bridge.InvokeRoute<string>(sendRoute, new string[] { JsonConvert.SerializeObject(Input) });
+            }
+            else
+            {
+                var ethSendTx = new EthSendTransaction(ThirdwebManager.Instance.SDK.session.Web3.Client);
+                return await ethSendTx.SendRequestAsync(Input);
+            }
+        }
+
+        private async Task<string> SendGasless()
+        {
+            if (Utils.IsWebGLBuild())
+            {
+                throw new UnityException("This functionality is not yet available on your current platform.");
+            }
+            else
+            {
+                string relayerUrl = ThirdwebManager.Instance.SDK.session.Options.gasless.Value.openzeppelin?.relayerUrl;
+                string forwarderAddress = ThirdwebManager.Instance.SDK.session.Options.gasless.Value.openzeppelin?.relayerForwarderAddress;
+                string forwarderDomain = ThirdwebManager.Instance.SDK.session.Options.gasless.Value.openzeppelin?.domainName;
+                string forwarderVersion = ThirdwebManager.Instance.SDK.session.Options.gasless.Value.openzeppelin?.domainVersion;
+
+                Input.Nonce = (
+                    await TransactionManager.ThirdwebRead<MinimalForwarder.GetNonceFunction, MinimalForwarder.GetNonceOutputDTO>(
+                        forwarderAddress,
+                        new MinimalForwarder.GetNonceFunction() { From = Input.From }
+                    )
+                ).ReturnValue1.ToHexBigInteger();
+
+                var request = new MinimalForwarder.ForwardRequest()
+                {
+                    From = Input.From,
+                    To = Input.To,
+                    Value = Input.Value,
+                    Gas = Input.Gas.Value,
+                    Nonce = Input.Nonce.Value,
+                    Data = Input.Data
+                };
+
+                var signature = await EIP712.GenerateSignature_MinimalForwarder(forwarderDomain, forwarderVersion, Input.ChainId, forwarderAddress, request);
+
+                var postData = new RelayerRequest(request, signature, forwarderAddress);
+
+                using (UnityWebRequest req = UnityWebRequest.Post(relayerUrl, ""))
+                {
+                    byte[] bodyRaw = System.Text.Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(postData));
+                    req.uploadHandler = (UploadHandler)new UploadHandlerRaw(bodyRaw);
+                    req.downloadHandler = (DownloadHandler)new DownloadHandlerBuffer();
+                    req.SetRequestHeader("Content-Type", "application/json");
+                    await req.SendWebRequest();
```

Comment:
the gasless code is re-used with the transaction manager no? might be useful to consolidate